.github/CONTRIBUTING.md

```diff
@@ -1,6 +1,6 @@
 # Contributing Guidelines
 
--   You can join us on [AnitaB.org Open Source Zulip](https://anitab-org.zulipchat.com/). Each active repo has its own stream to direct questions to (for example #powerup or #portal). Open Source Programs stream is [#open-source-progs](https://anitab-org.zulipchat.com/#streams/237907/open-source-progs).
+-   You can join us on [AnitaB.org Open Source Zulip](https://anitab-org.zulipchat.com/). Each active repo has its own stream to direct questions to (Each active repo has its own stream to direct questions (labeled as #question) on Zulip). Open Source Programs stream is [#open-source-progs](https://anitab-org.zulipchat.com/#narrow/stream/237907-open-source-progs).
```

Comment:
@The-Keshav-Agarwal I repeated the sentence to make it more clear how that needs to be framed. I guess you missunderstood and replaced it resulting `Each active repo has its own stream to direct questions` twice 😅.
The complete sentence should be like `Each active repo has its own stream to direct questions (labeled as #question) on Zulip)` only. If you have any queries do let us know :)

Reply:
@keshakaneria I corrected my mistake. please review it again.